### ISSUE TITLE
Can’t build on 32 bits Linux?

### ISSUE THREAD

**Issue by Hibou57** [protegeproject/protege#696]
I don’t know if it's related, but since the error message talks about compiler, I guess it may be. I wanted to build Protégé from source since the downloaded ZIP file seems to be 64 bits only. Building ends with this error:

    [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.3:compile (default-compile) on project protege-launcher: Fatal error compiling: invalid target release: 1.8 -> [Help 1]
    [ERROR] 
    [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
    [ERROR] Re-run Maven using the -X switch to enable full debug logging.
    [ERROR] 
    [ERROR] For more information about the errors and possible solutions, please read the following articles:
    [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
    [ERROR] 
    [ERROR] After correcting the problems, you can resume the build with the command
    [ERROR]   mvn <goals> -rf :protege-launcher

**Comment by ykazakov** [comment]
which version of java do you have?

`java -version`

I think, the latest Protege requires Java 1.8. Hence the error message.

**Comment by Hibou57** [comment]
You’re lovely, that was it.

I’m running Ubuntu 14.4 and it uses Java 7 instead of Java 8 which I had to install from a ppa.

If some people face the same issue, the PPA I used is `ppa:openjdk-r/ppa`. Then I did `sudo update-alternatives --config java` and selected Java 8.

As two side notes, there seems to be an issue with a pre‑installed plugin named `org.protege.editor.owl.rdf-2.0.2.jar` which I disable renaming it as `org.protege.editor.owl.rdf-2.0.2.jar.old` (installed Snap SPARQL instead after some web search). Then, if there remains an error about being unable to generate an ontology catalogue, one may have to properly set a field in File -> Preferences -> New ontology -> Default ontology IRI base.

( I’m closing the issue on the way :p )
